featuretools/tests/entityset_tests/test_pandas_es.py

```diff
@@ -716,6 +716,18 @@ def test_last_time_index(self, entityset):
         regions_lti = es["regions"].last_time_index.sort_index()
         assert (regions_lti == region_series.sort_index()).all()
 
+    def test_secondary_time_index(self, entityset):
+        es = entityset
+        es.normalize_entity('log', 'values', 'value',
+                            make_time_index=True,
+                            make_secondary_time_index={'datetime': []},
```

Comment:
This should be a dictionary mapping a date column to be used as the secondary time index column, to all the columns that became "known" or were recorded in the data at that time. To test this functionality (which I can see would not get handled properly) you should include an additional column to get added to the secondary time index of the new entity.

Reply:
Good point. I was using the `secondary_time_index` to create a "time_last" time index, but I'll pull along other columns in the test.